config/jobs/etcd/etcd-presubmits.yaml

```
@@ -236,3 +236,44 @@ presubmits:
           limits:
             cpu: "4"
             memory: "8Gi"
+
+  - name: pull-etcd-robustness-amd64
+    cluster: k8s-infra-prow-build
+    optional: true # remove this once the job is green
+    always_run: true
+    branches:
+      - main
+    decorate: true
+    decoration_config:
+      timeout: 60m
+    annotations:
+      testgrid-dashboards: sig-etcd-robustness
```

Let's show this job on both dashboards as suggested above by @ivanvc.

```suggestion
      testgrid-dashboards: sig-etcd-presubmits, sig-etcd-robustness
```

---

Yupp, adding it to both. Thank you!

---

Done 👍